console/src/assets/i18n/de.json

```diff
@@ -620,58 +620,58 @@
             }
         },
         "NAME": "Name",
-        "TYPE":"Applikationstyp",
-        "GRANT":"Grant Types",
+        "TYPE":"Anwendungstyp",
+        "GRANT":"Berechtigungstypen",
         "OIDC": {
             "PROSWITCH":"Konfigurator überspringen",
             "NAMEANDTYPESECTION":"Name und Typ",
-            "TITLEFIRST":"Geben Sie zuerst einen Namen ein!",
-            "TYPETITLE":"Welche Art von Applikation möchtest du erstellen?",
-            "REDIRECTTITLE":"Wohin soll nach dem Login weitergeleitet werden?",
-            "REDIRECTDESCRIPTIONWEB":"Der redirect muss mit https:// beginnen. http:// ist nur im Dev Mode zulässig!",
-            "REDIRECTDESCRIPTIONNATIVE":"Die Weiterleitung muss mit einem eigenen Protokoll oder http://localhost beginnen!",
-            "REDIRECTNOTVALID":"Diese Weiterleitung ist nicht zulässig!",
-            "POSTREDIRECTTITLE":"Das ist die Weiterleitung nach einem logout.",
-            "TYPEREQUIRED":"Der Typ ist notwendig!",
-            "TITLE": "OIDC Konfiguration",
+            "TITLEFIRST":"Gebe zuerst einen Namen ein.",
+            "TYPETITLE":"Welche Art von Anwendung möchtest Du erstellen?",
+            "REDIRECTTITLE":"Wohin soll nach dem Log-in weitergeleitet werden?",
+            "REDIRECTDESCRIPTIONWEB":"Die Weiterleitung muss mit https:// beginnen. http:// ist nur im Entwicklermodus zulässig.",
+            "REDIRECTDESCRIPTIONNATIVE":"Die Weiterleitung muss mit einem eigenen Protokoll oder http://localhost beginnen.",
+            "REDIRECTNOTVALID":"Diese Weiterleitung ist nicht zulässig.",
+            "POSTREDIRECTTITLE":"Das ist die Weiterleitung nach einem Log-out.",
+            "TYPEREQUIRED":"Der Typ ist notwendig.",
+            "TITLE": "OIDC-Konfiguration",
             "CLIENTID": "Client ID",
             "CLIENTSECRET": "Client Secret",
-            "CLIENTSECRET_DESCRIPTION": "Speichere das Client Secret, da es verschwindet, sobald der Dialog geschlossen wird!",
+            "CLIENTSECRET_DESCRIPTION": "Verwahren Sie Ihr Client Secret an einem sicheren Ort, da es nicht mehr angezeigt werden kann, sobald der Dialog geschlossen wird.",
```

Comment:
```suggestion
            "CLIENTSECRET_DESCRIPTION": "Verwahren das Client Secret an einem sicheren Ort, da es nicht mehr angezeigt werden kann, sobald der Dialog geschlossen wird.",
```

Reply:
Change `Verwahren` to `Verwahre`.

Reply:
👍 this is addressed here https://github.com/caos/zitadel/pull/745#discussion_r490911129